bedrock/exp/views.py

```diff
@@ -24,3 +28,29 @@ def new(request):
     return l10n_utils.render(
         request, 'exp/firefox/new/download.html', {'experience': experience, 'v': variant, 'active_locales': ['en-US', 'en-GB', 'en-CA', 'de']}
     )
+
+
+def home_view(request):
+    locale = l10n_utils.get_locale(request)
+    donate_params = settings.DONATE_PARAMS.get(
+        locale, settings.DONATE_PARAMS['en-US'])
+
+    # presets are stored as a string but, for the home banner
+    # we need it as a list.
+    donate_params['preset_list'] = donate_params['presets'].split(',')
+    ctx = {
+        'donate_params': donate_params,
+        'pocket_articles': PocketArticle.objects.all()[:4]
```

Comment:
I think you need to set `active_locales` in the context here seeing as these templates don't rely on a .lang file. See this view as an example: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/views.py#L791